Galegroup.js

```diff
@@ -2,20 +2,20 @@
 	"translatorID": "4ea89035-3dc4-4ae3-b22d-726bc0d83a64",
 	"label": "Galegroup",
 	"creator": "Sebastian Karcher and Aurimas Vinckevicius",
-	"target": "^https?://(find|go)\\.galegroup\\.com",
+	"target": "^https?://(find\\.galegroup\\.com/|go\\.galegroup\\.com/gdsc)",
 	"minVersion": "3.0",
 	"maxVersion": "",
 	"priority": 100,
```

Comment:
That's why there's additional logic in the Gale Databases translator on detect: it won't detect for gdsc because that doesn't have the citation tools. I don't think it's a problem if two targets with the same priority match as long as only one of the translators detects.